src/main/java/com/icssociety/automatedrequests/Main.java

```diff
@@ -54,6 +54,22 @@ public static void main(String[] args) throws HarReaderException {
 			
 			request.saveIt(); // saves it to the database
 		}
+
+		for(int i = 0; i < har.getLog().getEntries().size(); i++) { // loops through all the entries of the .har file
+			String req_id = har.getLog().getEntries().get(i).getRequest().getId().toString(); // request id
```

Comment:
This is not the correct id that we need, it is the request id from the database. This is going to be a bit more complicated as Ben was saying in Signal with foreign keys (still deciding whether or not we should do foreign keys). For now though, you can leave this and I'll figure out how we can get the request ids.